docs-parts/computation/06-distributed-computing_jobs_by_key.rst

```diff
@@ -3,17 +3,18 @@ This can be done by using `dj.key_hash` to convert the key as follows:
 
 .. code-block:: python
 
-    In [4]: schema.jobs & {'key_hash' : dj.key_hash({'id': 2})}                
-    Out[4]: 
+    In [4]: jk = {'table_name': '__job_results', 'key_hash' : dj.key_hash({'id': 2})}
```

Comment:
```suggestion
    In [4]: jk = {'table_name': JobResults.table_name, 'key_hash' : dj.key_hash({'id': 2})}
```

Reply:
good call.. saw this and was a bit 'hmm' about it.. will adjust.